### PR TITLE
Feat/export pfb to workspace

### PR DESCRIPTION
**Pull request by ZakirG**
This PR adds a new button to the explorer in Windmill labeled "Export PFB to Workspace" that requests Pelican to place a cohort PFB in indexd and then sends that PFB GUID to the manifestservice. 

Deployed in https://zakir.planx-pla.net/explorer

NOTE: The package.json will need to point to a new version of gen3-ui-component before merge. https://github.com/uc-cdis/gen3-ui-component/pull/79

The change was called for by this ticket https://ctds-planx.atlassian.net/browse/PXP-5895

and specified by this design doc:
https://docs.google.com/document/d/1954SSzVgDDM41uBksIt9RY7_MRAcCzoaMfmwHtfpFG4/edit#

### New Features
- New button in Windmill, "Export PFB to Workspace", which requests Pelican to place a cohort PFB in indexd and then sends that PFB GUID to the manifestservice. 

### Deployment changes
- A new Export to PFB button can be optionally enabled in the config.json. It is disabled by default. The button requires at least pelican:2020.10 release.

### CODE REVIEW

**Comment by mpingram** [comment]
Can you add some documentation on what the button does, how to configure it, and a link to the design doc to the `docs` folder?

**Comment by mpingram** [comment]
The docs look great 🙏

**Comment by mpingram** [comment]
Oh, one more thing, once you know what the necessary fuse-sidecar / manifestservice / pelican service versions are, can you add them to the `Deployment Changes` section of the PR notes?

**Comment by ZakirG** [comment]
> Oh, one more thing, once you know what the necessary fuse-sidecar / manifestservice / pelican service versions are, can you add them to the `Deployment Changes` section of the PR notes?

Yes, will do!

**Review by mpingram — APPROVED** [review]

File: src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx [hunk 1]
```diff
@@ -414,13 +428,47 @@ class ExplorerButtonGroup extends React.Component {
...
+  }
+
   sendPFBToSevenBridges = () => {
     const url = encodeURIComponent(this.state.exportPFBURL);
     window.location = `${this.props.buttonConfig.sevenBridgesExportURL}?format=PFB&url=${url}`;
   }
 
   exportToPFB = () => {
-    this.props.submitJob({ action: 'export', input: { filter: getGQLFilter(this.props.filter) } });
+    this.props.submitJob({ action: 'export', access_format: 'presigned_url', input: { filter: getGQLFilter(this.props.filter) } });
```
[the remark this answers is not in the document]
Comment (ZakirG) [reply src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx]: This change is not strictly necessary, as pelican 2020.10 defaults that argument to "presigned_url". I had included it to be explicit. I'll remove it to be on the safe side.

**Pull request merged**